src/inference/mh.jl

```diff
@@ -120,7 +121,7 @@ end
 """
     (new_trace, accepted) = mh(trace, selection::Selection; ..)
     (new_trace, accepted) = mh(trace, proposal::GenerativeFunction, proposal_args::Tuple; ..)
-    (new_trace, accepted) = mh(trace, proposal::GenerativeFunction, proposal_args::Tuple, involution::Function; ..)
```

Comment:
What is `..` here?

Reply:
The `..` stands for the keyword arguments, which are:
`check=false, observations=EmptyChoiceMap()`